src/register/RegistrationFields/CountryField/CountryField.jsx

```diff
@@ -65,30 +71,27 @@ const CountryField = (props) => {
 
     const { value } = event.target;
 
-    const { countryCode, displayValue, error } = validateCountryField(
+    const { error } = validateCountryField(
       value.trim(), countryList, formatMessage(messages['empty.country.field.error']), formatMessage(messages['invalid.country.field.error']),
     );
-
-    onChangeHandler({ target: { name: 'country' } }, { countryCode, displayValue });
```

Comment:
why is this removed?

Reply:
Now we are calling this onChangeHandler inside the onChange function and it does not make sense to keep it here inside the blur event